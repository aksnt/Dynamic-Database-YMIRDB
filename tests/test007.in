SET a 1 2
SNAPSHOT
SET b 4 5 6
SNAPSHOT
LIST ENTRIES
LIST KEYS
LIST SNAPSHOTS
ROLLBACK 1
LIST ENTRIES
LIST KEYS
LIST SNAPSHOTS
BYE
