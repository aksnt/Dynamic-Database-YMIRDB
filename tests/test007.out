> ok

> saved as snapshot 1

> ok

> saved as snapshot 2

> b [4 5 6]
a [1 2]

> b
a

> 2
1

> ok

> a [1 2]

> a

> 1

> bye
